assets/admin/editor-wizard/steps/course-patterns-step.js

```diff
@@ -1,43 +1,39 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
 /**
  * Internal dependencies
  */
-import PatternsList from '../patterns-list';
+import PatternsStep from './patterns-step';
 
 /**
- * Final step in course creation wizard choosing the actual course pattern to use.
+ * Course patterns step.
  *
- * @param {Object}   props
- * @param {Object}   props.data
- * @param {Function} props.setData
- * @param {Function} props.onCompletion
+ * @param {Object} props      Component props.
+ * @param {Object} props.data Wizard data.
  */
-/* eslint-disable no-unused-vars */
-const CoursePatternsStep = ( {
-	data: wizardData,
-	setData: setWizardData,
-	onCompletion,
-} ) => {
-	// TODO Implement this.
+const CoursePatternsStep = ( { data, ...props } ) => {
+	const replaces = {};
 
-	// We can call `onCompletion` to complete the wizard after setting the correct pattern with `setData`.
-	// We could replace `onCompletion` with the `goToNextStep` callback with a similar effect.
+	if ( data.courseTitle ) {
```

Comment:
When we implement the final patterns, if we don't have a course title to replace, we can remove this condition.